code/modules/modular_computers/hardware/job_disk.dm

```diff
@@ -229,3 +229,58 @@
 	name = "\improper FACEBUCKS disk"
 	icon_state = "cart-signal" // might need a new sprite
 	spam_delay = 5
+
+// maint spawn disks (including FACEBUCKS disk, and viruses)
+// Some of these are possibly inferior than its original version
+/obj/item/computer_hardware/hard_drive/role/random_maint_spawn
+	name = "random job disk"
+
+/obj/item/computer_hardware/hard_drive/role/random_maint_spawn/Initialize(mapload)
+	..()
+	var/static/disk_list = list(
+		/obj/item/computer_hardware/hard_drive/role/maint, // putting the same ones to make it higher spawn chance
+		/obj/item/computer_hardware/hard_drive/role/maint,
+		/obj/item/computer_hardware/hard_drive/role/maint,
+		/obj/item/computer_hardware/hard_drive/role/maint,
+		/obj/item/computer_hardware/hard_drive/role/maint,
+		/obj/item/computer_hardware/hard_drive/role/medical,
+		/obj/item/computer_hardware/hard_drive/role/chemistry,
+		/obj/item/computer_hardware/hard_drive/role/signal/toxins,
+		/obj/item/computer_hardware/hard_drive/role/security,
+		/obj/item/computer_hardware/hard_drive/role/quartermaster,
+		/obj/item/computer_hardware/hard_drive/role/virus/clown,
+		/obj/item/computer_hardware/hard_drive/role/virus/mime,
+		/obj/item/computer_hardware/hard_drive/role/virus/syndicate
```

Comment:
Do not give people this, maint detomatix sounds like a recipe for disaster

Reply:
It is 0 charge